test/protractor.conf.js

```diff
@@ -172,10 +174,10 @@ exports.config = {
   // The timeout in milliseconds for each script run on the browser. This should
   // be longer than the maximum time your application needs to stabilize between
   // tasks.
-  allScriptsTimeout: 11000,
+  allScriptsTimeout: 30 * 1000,//11000,
```

Comment:
Yeah I'm fine not adding the 30s comment because it's easy to forget to fix those when the code changes